changelog.json

```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "1.5.0-beta2": [
+      "[New] Successful merge banner - #5851",
```

Comment:
Isn't this `[Added]` and not new?

Reply:
I thought we used `[Added]` things like text editors and code highlighting?